GVFS/GVFS.UnitTests/Common/NuGetUpgrade/NuGetUpgraderTests.cs

```diff
@@ -320,12 +320,13 @@ public void TestUpgradeAllowed()
         [TestCase]
         public void WellKnownArgumentTokensReplaced()
         {
+            string logDirectory = ProductUpgraderInfo.GetLogDirectoryPath();

```

Comment:
Switch this to using a `mock:\` path instead of `ProductUpgraderInfo.GetLogDirectoryPath`? 

In my experience using mock paths helps prevent us from accidentally hitting the disk during unit tests (and helps find spot's we missed using `PhysicalFileSystem` in the product code).